batch/models.py

```diff
@@ -24,13 +24,17 @@ def get_queryset(self):
         return super().get_queryset().select_related("file")
 
 
+PROJECT_WORKFLOWS = ("Pixel Classification", "Object Classification")
```

Comment:
I think here snake_case should be used. While I know that in ilastik workflow name is used in this format and relied upon, I think can be easily misinterpreted as a human-readable string and changed. Related https://github.com/ilastik/ilastik/pull/2219

Reply:
See question/discussion in related.